Makefile

```diff
@@ -683,6 +684,20 @@ test-capd-junit: $(SETUP_ENVTEST) $(GOTESTSUM) ## Run unit and integration tests
 	$(GOTESTSUM) --junitfile $(ARTIFACTS)/junit.infra_docker.xml --raw-command cat $(ARTIFACTS)/junit.infra_docker.stdout
 	exit $$(cat $(ARTIFACTS)/junit.infra_docker.exitcode)
 
+.PHONY: test-runtime-sdk-test-extension
+test-runtime-sdk-test-extension: $(SETUP_ENVTEST) ## Run unit and integration tests for the test extension
+	cd $(TEST_EXTENSION_DIR); KUBEBUILDER_ASSETS="$(KUBEBUILDER_ASSETS)" go test ./... $(TEST_ARGS)
+
+.PHONY: test-runtime-sdk-test-extension-verbose
+test-runtime-sdk-test-extension-verbose: ## Run unit and integration tests with verbose flag
+	$(MAKE) test-runtime-sdk-test-extension TEST_ARGS="$(TEST_ARGS) -v"
+
+.PHONY: test-runtime-sdk-test-extension-junit
+test-runtime-sdk-test-extension-junit: $(SETUP_ENVTEST) $(GOTESTSUM) ## Run unit and integration tests and generate a junit report for the test extension
```

Comment:
Added junit (as in the CAPD target) though I don't know how useful it might be. We could easily remove this one, or dedupe if needed.